src/test/java/com/example/account/repository/impl/ElascticsearchContainerConfig.java

```diff
@@ -1,18 +1,22 @@
-package com.example.account.elastic.repository.impl;
+package com.example.account.repository.impl;
 
 import org.testcontainers.elasticsearch.ElasticsearchContainer;
 import org.testcontainers.utility.DockerImageName;
 
 public class ElascticsearchContainerConfig extends ElasticsearchContainer {
```

Comment:
@amerkul read about `composition vs. inheritance`
it's [here](https://stackoverflow.com/questions/49002/prefer-composition-over-inheritance)

Reply:
I am afraid of you

Reply:
@amerkul yeah, I'm aware of

Reply:
Too much answers -> it's [here](https://stackoverflow.com/questions/49002/prefer-composition-over-inheritance)